cargo/spec/fixtures/projects/version_conflict/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "version_conflict"
+version = "0.1.0"
+authors = ["support@dependabot.com"]
+
+[dependencies]
+askama = { git = "https://github.com/djc/askama", branch = "main" }
+criterion = "0.3"
+handlebars = { git = "https://github.com/sunng87/handlebars-rust" }
+horrorshow = { git = "https://github.com/Stebalien/horrorshow-rs" }
+liquid = { git = "https://github.com/cobalt-org/liquid-rust" }
+markup = { git = "https://github.com/utkarshkukreti/markup.rs" }
+sailfish = { git = "https://github.com/Kogia-sima/sailfish" }
+sailfish-macros = { git = "https://github.com/Kogia-sima/sailfish" }
+serde = "1"
+serde_derive = "1"
+serde_json = "1"
+serde_yaml = "0.8"
+tera = { git = "https://github.com/Keats/tera" }
+fomat-macros = { git = "https://github.com/krdln/fomat-macros" }
+maud = { git = "https://github.com/lambda-fairy/maud" }
+
+[build-dependencies]
+ructe = { git = "https://github.com/kaj/ructe" }
```

Comment:
Wondering, could we cut this down to just `ructe` and `askma` which seem to have the conflicting deps? Had a go at testing this locally and the first time the test ran it took over 2 mins to run so might be due to the large poject

Reply:
Ah, yes. I remember you mentioned that we should whittle this down to just the important bits. I'll try to reduce this and the .lock file to just the relevant bits.

Reply:
Nice one, you might be able to just edit the `Cargo.toml` and run [`cargo generate-lockfile`](https://doc.rust-lang.org/cargo/commands/cargo-generate-lockfile.html)